vaultpress.php

```diff
@@ -306,7 +306,7 @@ function load_menu() {
 		if ( class_exists( 'Jetpack' ) ) {
 			$hook = add_submenu_page( 'jetpack', 'VaultPress', 'VaultPress', 'manage_options', 'vaultpress', array( $this, 'ui' ) );
 		} else {
-			$hook = add_menu_page( 'VaultPress', 'VaultPress', 'manage_options', 'vaultpress', array( $this, 'ui' ), 'div' );
+			$hook = add_menu_page( 'VaultPress1', 'VaultPress', 'manage_options', 'vaultpress', array( $this, 'ui' ), 'div' );
```

Comment:
Is this `VaultPress1` change intentional?

Reply:
Nope. I must've typed it somehow.